pkg/services/annotations/annotationsimpl/composite_store.go

```diff
@@ -2,12 +2,14 @@ package annotationsimpl
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"sort"
 
 	"github.com/grafana/grafana/pkg/infra/log"
 
 	"github.com/grafana/dskit/concurrency"
+
```

Comment:
?

Reply:
🤦‍♂️ thanks - I meant to keep an eye on the goimports shenanigans!